### DISCUSSION TITLE
App Store Connect API key Error

### DISCUSSION THREAD

**Discussion by OM-Tecnologia** [codemagic-ci-cd/codemagic-docs#2046] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Create a new Key on the App Store Connect
On Codemagic go to Teams - Personal Account - Integrations - Developer Portal - Manage Keys and include the new Key

Apps - Workflow Editor - Distribution - iOS code signing
Select Automatic
Select the new key included on "App Store Connect API key"

Error returned:
```
Unable to load bundle identifiers

Loading bundle identifiers for OMTecnologiaParaNegocios11 failed. A required agreement is missing or has expired. - This request requires an in-effect agreement that has not been signed or has expired.
```

### Expected results

No errors, and be able to select the Bundle ID

### Actual results

```
Unable to load bundle identifiers

Loading bundle identifiers for OMTecnologiaParaNegocios11 failed. A required agreement is missing or has expired. - This request requires an in-effect agreement that has not been signed or has expired.
```

**Comment by dtrdic** [comment, marked as answer]
hey @OM-Tecnologia, sounds like your Apple Developer account needs some updates in regards to the agreements and licenses. Try checking here https://appstoreconnect.apple.com/agreements/#/
